pygsti/tools/matrixtools.py

```diff
@@ -904,9 +955,13 @@ def norm1(m):
     -------
     numpy.ndarray
     """
-    return float(_np.real(_np.trace(_sqrtm(_np.dot(m.conj().T, m)))))
+    s = _spl.svdvals(m)
+    nrm = _np.sum(s)
+    return nrm
```

Comment:
The new implementation is asymptotically faster if ``m`` is wide. In all other situations the asymptotic runtime is unchanged, but the constant factors are better for computation and memory use.